synfig-studio/src/synfigapp/actions/layerlowermax.cpp

```diff
@@ -0,0 +1,157 @@
+/* === S Y N F I G ========================================================= */
+/*!	\file layerlowermax.cpp
+**	\brief Template File
+**
+**	\legal
+**	Copyright (c) 2002-2005 Robert B. Quattlebaum Jr., Adrian Bentley
+**	Copyright (c) 2008 Chris Moore
```

Comment:
maybe add 
**	Copyright (c) 2023 Synfig Contributors?

I'm not sure... :P

Reply:
Sure I think it wouldn't hurt to add it although I'm not too sure either.

Reply:
Done